packages/editor-ui/src/event-bus/mfa.ts

```diff
@@ -7,8 +7,10 @@ export interface MfaModalClosedEventPayload {
 }
 
 export interface MfaModalEvents {
+	/** Command to request closing of the modal */
 	close: MfaModalClosedEventPayload | undefined;
```

Comment:
Should this instead be `close: MfaModalClosedEventPayload`, and `mfaCode` instead be `?: string` ?

Reply:
I wanted to make a clear distinction between when the value is coming from the MFA modal and when from the "plumbing" (i.e. the `Modal` base). `undefined` means the user closed the modal (handled by `Modal`), a value means `PromptMfaCodeModal` closed the modal.